### ISSUE TITLE
Allow passing a message to be set in the TapiocaException through ResponseProcessException

### ISSUE THREAD

**Issue by filipeximenes** [vintasoftware/tapioca-wrapper#95]

**Comment by tarsisazevedo** [comment]
We can get this message from a `error` field at response body and send it to tapioca exception. What do you think?

**Comment by filipeximenes** [comment]
@tarsisazevedo Yes, the default action may be look for an `error` field in the body as this is probably the most common way to pass error messages. But this should not fail if the field is not present.
